rest/data/interface.go

```diff
@@ -407,15 +407,9 @@ type MockConnector interface {
 	// if bool is true, move the commit queue item to be processed next.
 	EnqueueItem(string, restModel.APICommitQueueItem, bool) (int, error)
 	AddPatchForPr(ctx context.Context, projectRef model.ProjectRef, prNum int, modules []restModel.APIModule, messageOverride string) (string, error)
-	FindCommitQueueForProject(string) (*restModel.APICommitQueue, error)
-	EnableCommitQueue(*model.ProjectRef) error
-	CommitQueueRemoveItem(string, string, string) (*restModel.APICommitQueueItem, error)
-	IsItemOnCommitQueue(string, string) (bool, error)
-	CommitQueueClearAll() (int, error)
 	CreatePatchForMerge(context.Context, string, string) (*restModel.APIPatch, error)
-	IsPatchEmpty(string) (bool, error)
 	IsAuthorizedToPatchAndMerge(context.Context, *evergreen.Settings, UserRepoInfo) (bool, error)
-	GetMessageForPatch(string) (string, error)
-	ConcludeMerge(string, string) error
-	GetAdditionalPatches(patchId string) ([]string, error)
+	HasMatchingGitTagAliasAndRemotePath(string, string) (bool, string, error)
+	GetProjectFromFile(context.Context, model.ProjectRef, string, string) (model.ProjectInfo, error)
+	CreateVersionFromConfig(context.Context, *model.ProjectInfo, model.VersionMetadata, bool) (*model.Version, error)
```

Comment:
I'm also not 100% certain why this needs to be mocked; in the existing codebase the only tests I see that use this don't use the mock version.

Reply:
This is the only case where we are creating a git tagged version along with it, which requires authenticating with Github

Reply:
in TestCreateVersionForTag